packages/common/client-utils/src/generateNames.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import generateStupidName from "sillyname";
+
+/**
+ * Utility function for generating a random name.
+ *
+ * @internal
+ */
+export function getRandomName(connector = "_", capitalize = false): string {
```

Comment:
I am wondering if this is useful for this functionality to be shared?  No other places is using it, should we just put it with the t9s-driver?

Reply:
As you mentioned, I am currently working on replacing the dependencies for other packages too. Thank you for the feedback 😄 .